delocate/delocating.py

```diff
@@ -450,6 +450,37 @@ def _merge_lib_dict(d1, d2):
     return None
 
 
+def _decide_dylib_bundle_directory(
+    wheel_dir: str, package_name: str, lib_sdir: str = ".dylibs"
+) -> str:
+    """Return a relative directory which should be used to store dylib files.
+
+    Parameters
+    ----------
+    wheel_dir : str
+        The directory of an unpacked wheel to analyse.
+    package_name : str
+        The name of the package.
+    lib_sdir : str, optional
+        This value passed in via :func:`delocate_wheel`.
+
+    Returns
+    -------
+    dylibs_dir : str
+        A path to within `wheel_dir` where any library files should be put.
+    """
+    package_dirs = find_package_dirs(wheel_dir)
+    for directory in package_dirs:
+        if directory.endswith(package_name):
+            # Prefer using the directory with the same name as the package.
+            return pjoin(directory, lib_sdir)
+    if package_dirs:
```

Comment:
Just a personal preference so I don't have to think about what True might mean in the bare case:

```suggestion
    if len(package_dirs):
```

Reply:
I never got confused by this truth test.  It's handled consistently across Python that sequences are True when they have any items and False when empty. PEP8 also recommended not adding `len` to sequences, which is likely why I got used to doing it this way.

Reply:
Your call.  For me, I'm thinking - could this be None?  Or an integer?

Reply:
If your IDE supports type hinting then it will plainly tell you that `package_dirs` is `Set[str]`.  If the IDE only shows the docstrings of functions then looking at `find_package_dirs` will tell you that it returns `Set[str]`.  Writing code with mystery types should become a thing of the past.

`None` and `int` are ruled out just from how the name is being used in the for loop above and in the `min` function within the if branch.  It's clearly a sequence, but without type hinting you wouldn't be able to tell if it was a sequence of `str` or something else, such as `pathlib.Path`.

Reply:
Yes, sure, I could use other information by looking around, or if I had the right IDE with type hints (I don't), but with `len(foo)` I know immediately by looking at it...